_inc/client/plans/single-product-backup.scss

```diff
@@ -1,3 +1,21 @@
+@import '../scss/variables/_colors.scss';
+
+.plans-section__header {
+	color: #23282d;
+	text-align: center;
+	font-family: Roboto, sans-serif;
```

Comment:
I don't think we should override the default font here. I can see that Jetpack consistently uses a default font. @jeherve Can you confirm?

Reply:
I think you're right. I had just been matching the design exactly, but I think it makes more sense to use the default font.